Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -4,22 +4,28 @@
 using GVR.Caches;
 using GVR.Demo;
 using GVR.Phoenix.Data;
-using GVR.Phoenix.Interface;
 using GVR.Phoenix.Util;
 using GVR.Util;
 using PxCs.Data.Struct;
 using PxCs.Data.Vm;
 using PxCs.Data.Vob;
-using PxCs.Interface;
 using UnityEngine;
+using UnityEngine.Rendering.Universal;
 using UnityEngine.XR.Interaction.Toolkit;
 using static PxCs.Interface.PxWorld;
 using Vector3 = System.Numerics.Vector3;
 
+#if UNITY_EDITOR
+    using UnityEditor;
+#endif
+
+
 namespace GVR.Creator
 {
 	public class VobCreator : SingletonBehaviour<VobCreator>
-    {   
+    {
+        private const float decalOpacity = 0.5f;
```

Comment:
decals should be 1f most of the time, exception should be maybe dirt on castle walls

Reply:
I would disagree. We discussed within Discord and mentioned 0.5f-0.75f
At least the walls look fine with this value and I would not want to have it "adjusted" for specific decals as of now.

Suggestion: Let's stick with a value and optimize it once we have lightning done right.